test/lib/transformers/transformersTest.js

```diff
@@ -18,7 +19,7 @@ describe('mapOutReq', () => {
       port: undefined,
       path: 'path',
       query: 'query',
-      href: undefined,
+      href: url.format(inMsg.uri),
```

Comment:
Whats the value of url.format(inMsg.uri) ? Maybe add the literal here

Reply:
We use `format` to create the url from the request options object. 
I preferred to have this in the test method in order not to update every test if anything changes (how the url is formatted) in that method in future node versions.

Reply:
If anything changes won't the logs of Riviere also change? Shouldn't tests catch that?

Reply:
Done